src/pages/chainlink-functions/resources/service-limits.md

```diff
@@ -6,20 +6,20 @@ title: "Chainlink Functions Service Limits"
-| Maximum HTTP request length (body + headers) | 2 kilobytes                                                                                                                     |
-| Maximum HTTP response length                 | 2 megabytes                                                                                                                     |
+| Item                                                          | Description                                                                                                    | Limits                                                                                                                          |
+| ------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| Supported languages                                           | Supported language of the Source code that is submitted in the Chainlink Functions request                     | Only JavaScript is supported for the moment. Your source code can only use vanilla Node.js features (you cannot import modules) |
+| Maximum requests in flight                                    | The maximum number of simultaneous Chainlink Functions requests that are not fulfilled yet by the DON          | Limited by the [effective balance](/chainlink-functions/resources/concepts#subscriptions)                                       |
+| Maximum callback gas limit                                    | The maximum amount of gas that you can set for Chainlink Functions to fulfill your callback function           | 300,000                                                                                                                         |
+| Maximum subscriptions                                         | Maximum subscriptions that you can create                                                                      | unbounded                                                                                                                       |
+| Maximum consumer contracts per subscription                   | The maximum number of consumer contracts that you can add to a subscription                                    | 100                                                                                                                             |
+| Request fulfillment timeout                                   | The maximum duration of an in-flight Chainlink Functions request. After this duration, the request is canceled | 5 minutes                                                                                                                       |
```

Comment:
"the request is canceled" is a bit confusing. Maybe say "After this duration, the request times out and your subscription won't be charged. You can request that locked funds are returned to your subscription balance."

... or something along those lines if you want to phrase is in a better way.